### PR TITLE
EN-95:  Set active field to Assignment entity

### PR DESCRIPTION
**Pull request by FedericoA15**
# Description :pencil:

The ability was implemented to display the active and inactive Assignments on the employee's profile, highlighting the active ones with the color green. Additionally, in the Assignment tab, only the active ones will be shown.

## Link to ticket :link:

https://entropyteam.atlassian.net/browse/EN-95

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? :microscope:

image

image

### CODE REVIEW

File: src/assignments.js
Comment (agustinb4) [inline src/assignments.js:1]: I think we could leverage this changes to migrate this file into type script. 
Please compare the syntax with contracts.tsx, then change the file extention to tsx. please check with @baremarco if other changes are needed.

**Review by FacuOrellana — APPROVED** [review]

File: src/assignments.js
Comment (FedericoA15) [reply src/assignments.js:1]: ready, there I adapted with typescript and I could also add the filter for active although I could not finish the part that by default I filtered them in active using filterbuttons.

File: src/components/filters/QuickFilter.tsx
Comment (agustinb4) [inline src/components/filters/QuickFilter.tsx:1]: the name of the file should be quickFilter

File: src/types/QuickFilterProps.ts
Comment (agustinb4) [inline src/types/QuickFilterProps.ts:1]: I would change the file name to quickFilterModel, also the interface name should be IQuickFilterModel

**Review by agustinb4 — APPROVED** [review]

**Pull request merged**